src/irmin-mirage/git/irmin_mirage_git.ml

```diff
@@ -261,30 +281,59 @@ module KV_RW (G : Irmin_git.G) (C : Mirage_clock.PCLOCK) = struct
   let info t op = Info.f ~author:(t.author ()) "%s" (t.msg op)
   let path = RO.path
 
+  let ( >?= ) r f =
+    r >>= function
+    | Error e -> Lwt.return_error (e :> write_error)
+    | Ok r -> f r
+
+  let lwt_ok _ = Lwt.return_ok ()
```

Comment:
Just a stylistic nit, but I'm not sure this is really needed and it seems to be used inconsistently. Some additions in this PR just use `Lwt.return_ok` directly.

I'd maybe just drop this shortcut.